src/proper_parenthetics.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+
+from stack import Stack
+
+
+def parenthetics(parentheses_string):
+    """Check for valid parenthetics."""
+    parentheses_stack = Stack()
+    for item in parentheses_string:
+        if item == '(':
+            parentheses_stack.push(item)
+        elif item == ')':
+            try:
+                parentheses_stack.pop()
+            except IndexError:
+                return -1
+        if len(parentheses_stack) == 0:
```

Comment:
I imagine that this line is causing some trouble with your tests.  Does your `Stack` implementation implement the `__len__` special method?  How does the `len` builtin work with your stack?